packages/legend-studio/src/models/metamodels/pure/model/packageableElements/domain/Association.ts

```diff
@@ -139,13 +139,14 @@ export class Association
       Class,
       `Association property '${property.name}' must be of type 'class'`,
     );
-    assertTrue(
-      deleteEntry(
-        otherPropertyAssociatedClass.propertiesFromAssociations,
-        otherProperty,
-      ),
-      `Can't find property '${otherProperty.name}' from association '${this.path}' in associated class '${otherPropertyAssociatedClass.path}'`,
-    );
+    otherPropertyAssociatedClass.propertiesFromAssociations.length &&
```

Comment:
can we use a normal `if` here?

Reply:
Also my concern here why is the `propertiesFromAssociations` of the other class is empty? Did we mess up building the graph at some point?